scipy/sparse/linalg/isolve/utils.py

```diff
@@ -120,4 +111,17 @@ def postprocess(x):
         if A.shape != M.shape:
             raise ValueError('matrix and preconditioner have different shapes')
 
+    # set initial guess
+    if x0 is None:
+        x = zeros(N, dtype=xtype)
+    elif x0 is True:  # use nonzero initial guess ``M * b``
```

Comment:
`True` is a bit ugly. I think if there's multiple built-in options, using string names would be better (and extensible in case another method needs adding). Is there a sensible string name you can think of for this?